src/Game/AccountManagement.java

```diff
@@ -41,45 +44,28 @@ public static void register(String user, String pass) {
 				int port = Replay.connection_port;
 				StreamUtil.initializeStream(Client.server_address, port);
 				StreamUtil.setStreamMaxRetries(Client.maxRetries);
-				byte limit30 = 0;
-				String param = Client.getParameter("limit30");
-				if (param != null && param.equals("1")) {
-					limit30 = 1;
-				}
 				
-				int[] keys = new int[]{
-						(int)(9.9999999E7 * Math.random()),
-						(int)(9.9999999E7 * Math.random()),
-						(int)(9.9999999E7 * Math.random()),
-						(int)(9.9999999E7 * Math.random())
-				};
 				StreamUtil.newPacket(2);

```

Comment:
Above this line insert
`int session_id = 0; // TODO: should read session ID here, triggered by TCP handshake`

Reply:
placed